src/families/bitcoin/react.ts

```diff
@@ -1,9 +1,10 @@
-import invariant from "invariant";
 import type { NetworkInfo, Transaction } from "./types";
 import type { FeeStrategy, Account } from "../../types";
 export const useFeesStrategy = (a: Account, t: Transaction): FeeStrategy[] => {
   const networkInfo = t.networkInfo;
-  invariant(networkInfo, "no network info");
+
+  if (!networkInfo) return [];
```

Comment:
this actually changes the behavior, wouldn't it be better with a 
```suggestion
  if (!networkInfo) throw new Error("no network info")
```

?